cmd/kubeapps-apis/docs/kubeapps-apis.swagger.json

```diff
@@ -3794,16 +3794,16 @@
     "v1alpha1SecretType": {
       "type": "string",
       "enum": [
-        "OPAQUE",
-        "SERVICE_ACCOUNT_TOKEN",
-        "DOCKER_CONFIG",
-        "DOCKER_CONFIG_JSON",
-        "BASIC_AUTH",
-        "SSH_AUTH",
-        "TLS",
-        "BOOTSTRAP_TOKEN"
+        "SECRET_TYPE_OPAQUE_UNSPECIFIED",
+        "SECRET_TYPE_SERVICE_ACCOUNT_TOKEN",
+        "SECRET_TYPE_DOCKER_CONFIG",
+        "SECRET_TYPE_DOCKER_CONFIG_JSON",
+        "SECRET_TYPE_BASIC_AUTH",
+        "SECRET_TYPE_SSH_AUTH",
+        "SECRET_TYPE_TLS",
+        "SECRET_TYPE_BOOTSTRAP_TOKEN"
```

Comment:
Adding `SECRET_TYPE_` as a prefix here produces code with redundant names like `SecretType_SECRET_TYPE_OPAQUE_UNSPECIFIED` (in `cmd/kubeapps-apis/gen/plugins/resources/v1alpha1/resources.pb.go`below).
Not a big problem at all, mainly readability. Couldn't we avoid it? Maybe it is a requirement for the API?

Reply:
Yeah, I know it feels it (and I did originally do it without the prefix), but then `buf lint` fails as this is one of their lint rules. They explain why here: https://docs.buf.build/lint/rules#enum_value_prefix